code/datums/traits/positive_quirk.dm

```diff
@@ -139,7 +139,8 @@
 //Credit To Yowii/Yoworii/Yorii for a much more streamlined method of language library building
 
 /datum/quirk/multilingual/add()
-	if(!known_language)
+	known_language = read_choice_preference(/datum/preference/choiced/quirk/multilingual_language)
+	if(!known_language) // default to random
```

Comment:
known_language will never be null as long as `prefs` exists, because it will always generate a default from the list (it will persist, as well)

Reply:
leftover code from my previous attempts at implementing a "random" option